<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATL-Electrical Chemistry Analysor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="positiveTab">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>22</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="positiveDataForm">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QTableWidget" name="tablePositiveData">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>12</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>16</number>
      </attribute>
      <column>
       <property name="text">
        <string>Voltage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="positiveQV">
     <attribute name="title">
      <string>Q-V</string>
     </attribute>
     <widget class="QFrame" name="frame">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="positiveDQDV">
     <attribute name="title">
      <string>dQ-dV</string>
     </attribute>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QTabWidget" name="positiveTab_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>351</width>
      <height>251</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="positiveDataForm_3">
     <attribute name="title">
      <string>Data</string>
     </attribute>
     <widget class="QTableWidget" name="tablePositiveData_3">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>351</width>
        <height>251</height>
       </rect>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::IgnoreAction</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>12</number>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>16</number>
      </attribute>
      <column>
       <property name="text">
        <string>Voltage</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Capacity</string>
       </property>
      </column>
     </widget>
    </widget>
    <widget class="QWidget" name="positiveQV_3">
     <attribute name="title">
      <string>Q-V</string>
     </attribute>
     <widget class="QFrame" name="frame_5">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>351</width>
        <height>241</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="positiveDQDV_3">
     <attribute name="title">
      <string>dQ-dV</string>
     </attribute>
     <widget class="QFrame" name="frame_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>261</width>
        <height>181</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>40</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_7">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>300</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>861</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="Chinese" country="China"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuLoad">
     <property name="title">
      <string>Open</string>
     </property>
     <addaction name="actionPositive_Reference"/>
     <addaction name="actionNegative_Reference"/>
     <addaction name="actionMeasured_Data"/>
    </widget>
    <addaction name="menuLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionPreference"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionPositive_Reference">
   <property name="text">
    <string>Positive Reference</string>
   </property>
  </action>
  <action name="actionNegative_Reference">
   <property name="text">
    <string>Negative Reference</string>
   </property>
  </action>
  <action name="actionMeasured_Data">
   <property name="text">
    <string>Measured Data</string>
   </property>
  </action>
  <action name="actionPreference">
   <property name="text">
    <string>Preferences...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
